### ISSUE TITLE
Using tabbed widget in changelist for editable fields.

### ISSUE THREAD

**Issue by fabiocaccamo** [deschler/django-modeltranslation#479]
Hi, 
is it possible to use the tabbed widget in changelist for editable fields (listed in `list_editable`)?

**Comment by fabiocaccamo** [comment]
@last-partizan have you any update about this issue?

**Comment by last-partizan** [comment]
@fabiocaccamo no, it is not possible.

**Comment by fabiocaccamo** [comment]
I don't think it is impossible: https://stackoverflow.com/questions/7065982/list-editable-and-widgets

**Comment by last-partizan** [comment]
Well, technically you can change widgets in that list.
But django-modeltranslation does not care about widgets, it's using jquery.ui and lot of magick to make it work at change_form.

Something similar can be achived at changelist, so you can try to make it work and create pull-request.

(In my opinion, jquery is really bad choice for this task and it should be rewritten as vuejs widgets, or maybe webcomponents)
